src/components/Collection/SaveToCollectionModal/SaveToCollectionModal.tsx

```diff
@@ -75,7 +77,7 @@ const SaveToCollectionModal = ({
                 ]}
                 actionText={t('common:submit')}
                 onSubmit={(data: any) => {
-                  onNewCollectionCreated(data.name);
+                  onNewCollectionCreated(data.name).then(() => setIsAddingNewCollection(false));
```

Comment:
What happens when there is an issue with the API? I think we should show a toast/error message